### ISSUE TITLE
Request - Accessible Tooltip Component

### ISSUE THREAD

**Issue by iamhollow** [abyssjs/abyss-ui#7]
**What is it:**
A flexible, customizable Tooltip that is accessibility compliant. The Tooltip element should be able to be Tab-focused, and opened/closed using the Up-Arrow and Down-Arrow keys. The Tooltip should be properly aria-labeled. It should use JSS styling and be compatible with abyss theming.

**Example:**
https://84vn36m178.codesandbox.io/
https://github.com/gilbarbara/react-floater

<img width="664" alt="Screen Shot 2019-09-30 at 2 34 52 PM" src="https://user-images.githubusercontent.com/7559688/65910032-7e483280-e38f-11e9-8275-4874edcf4682.png">

**Comment by lnaia** [comment]
If no one else minds, I can take this.

**Comment by n2lel** [comment]
my apologies for not closing, this is already merged in.
